test/dummy/test/controllers/application_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+class ApplicationControllerTest < ActionController::TestCase
+  test "truth" do
+    assert true
+  end
+
+  test "negativity" do
+    assert_not false
+  end
```

Comment:
I think that "Test LOC" will always be zero for `rake_test.rb` if this file is not present.

The "negativity" test is so that the ratio is not 1:1.